tests/test_builds.py

```diff
@@ -80,6 +102,24 @@ def test_get(client):
     assert response[0]['number'] == 1
 
 
+@responses.activate
+def test_get_selective(client):
+    responses.add(
+        responses.GET,
+        re.compile(r'.*/api/json'),
+        body=BUILDS_ALL_JSON_SELECTIVE,
+    )
+
+    fields = ['number', 'url']
+    response = client.builds.get('job', fields=fields, start=1, end=3)
+    assert len(responses.calls) == 1
+    requested_url = responses.calls[0].request.url
+    expected_url = 'http://server//job/job/api/json?tree=allBuilds%5Bnumber,url%5D%7B1,3%7D'
```

Comment:
it's incorrect, because we are pinned on `http://server`? Need to check url, please see the examples.

Moreover I suggest you to check all possible parameters
1) With only start
2) With only end
3) With start and end
4) Without it
5) Check that fields forms correctly